test/test_files/tinysnb/uuid/simpleTest.test

```diff
@@ -0,0 +1,22 @@
+-GROUP UUIDSimpleTest
+-DATASET CSV tinysnb
+
+--
+
+-CASE UUIDCreateInsertMatch
```

Comment:
Instead of creating a new test file, you can reuse the existing test files.
1. Return UUID('A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A11') => move to projection/single_label.test
2. CREATE (:Person {name: 'Alice', age: 25, u: 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A11'});
Move to update_test.